slog/slog.go

```diff
@@ -167,14 +168,14 @@ func (l *Logger) WithSpan(s *trace.Span) *Entry {
 	return l.entry().WithSpan(s)
 }
 
-// Label to be included as metadata for log entries.
-type Label struct {
+// Pair to be included as metadata for log entries.
+type Pair struct {
```

Comment:
Why the rename here, but keeping everything else named using "Label" ?

Reply:
`Pair` is shared between both `Labels` (which have supposed semantic meaning within logs, have their own namespace `logging.googleapis.com/labels` and are required to be string->string), and `Details`, which are free-form, included in the jsonPayload of the log entry and can represent any JSON literal or object.